src/lib/components/Modal/CollectionPopup.svelte

```diff
@@ -106,26 +106,26 @@
     class="d-flex align-items-center justify-content-end gap-3 mt-1 mb-0 rounded"
     style="font-size: 16px;"
   >
-    <CustomButton
+    <Button
       disable={deleteLoader}
-      text={"Cancel"}
-      fontSize={14}

```

Comment:
check if you can pass font size in rem/em unit and assign that value here or use the fs class from bootstrap